src/controllers/scorecontroller.js

```diff
@@ -0,0 +1,13 @@
+import Controller from './core/controller';
+
+
+export default class ScoreController extends Controller 
+{
+    updateScore()

```

Comment:
I think this can be static.

Reply:
(that'd solve the whole `(new ScoreController).updateScore()` stuff).

Just leave it for now, though.